inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBInstance.java

```diff
@@ -122,17 +122,18 @@ public void write(RepositoryConnection aConn, KnowledgeBase kb)
 
     public static KBInstance read(RepositoryConnection aConn, Statement aStmt)
     {
+        String language = "en";
         KBInstance kbInst = new KBInstance();
         kbInst.setType(URI.create(aStmt.getObject().stringValue()));
         kbInst.setIdentifier(aStmt.getSubject().stringValue());
         kbInst.originalStatements.add(aStmt);
 
-        readFirst(aConn, aStmt.getSubject(), RDFS.LABEL, null).ifPresent((stmt) -> {
+        readFirst(aConn, aStmt.getSubject(), RDFS.LABEL, null, language).ifPresent((stmt) -> {
```

Comment:
Is `RDFS.LABEL` a configurable IRI, e.g. `descrptionIRI`? What is the `null` here?

Reply:
It is not configurable.